tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/AuthorityTests.cs

```diff
@@ -88,6 +88,33 @@ public async Task FailedAuthorityValidationTestAsync()
             Assert.AreEqual("invalid_instance", exception.ErrorCode);
         }
 
+        [TestMethod]
+        public async Task AuthorityValidationTestWithFalseValidateAuthorityAsync()
+        {
+            LabResponse labResponse = await LabUserHelper.GetDefaultUserAsync().ConfigureAwait(false);
+            LabUser user = labResponse.User;
+
+            IPublicClientApplication pca = PublicClientApplicationBuilder
+                .Create(labResponse.App.AppId)
+                .WithAuthority("https://bogus.microsoft.com/common", false)
+                .WithTestLogging()
+                .Build();
+
+            Trace.WriteLine("Acquire a token using a not so common authority alias");
+
+            HttpRequestException exception = await AssertException.TaskThrowsAsync<HttpRequestException>(() =>
+                 pca.AcquireTokenByUsernamePassword(
+                    s_scopes,
+                     user.Upn,
+                     new NetworkCredential("", user.GetOrFetchPassword()).SecurePassword)
+                     .ExecuteAsync())
+                .ConfigureAwait(false);
+
+
+            Assert.IsTrue(((System.Net.WebException) exception.InnerException).Status == System.Net.WebExceptionStatus.NameResolutionFailure);
```

Comment:
@neha-bhargava - the PR build failed and it shows that a slightly different exception occurs on .NET Core. This is likely to do subtle differerences in the HTTP stack used by .NET classic (net45) and .NET core. I have a PR out to update the .netcore tests to 3.1 so that you can run them locally (well, it's a problem all of us are starting to have).